src/components/CreateCourseRunPage/index.jsx

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable camelcase */
```

Comment:
What in this file is causing us to need this disabled out of curiosity?

Reply:
Pulling out `course_runs` from the serialized data. Then `course_runs` is a JS variable, and eslint doesn't like that. We do same thing in main course form too.